src/main/java/com/googlecode/objectify/impl/LoadTypeImpl.java

```diff
@@ -122,6 +123,10 @@
 		for (S id: ids)
 			keymap.put(this.<T>makeKey(id), id);
 
+		if (keymap.isEmpty()) {
+			return Maps.newLinkedHashMap();

```

Comment:
TBH there is no proper consistency in the objectify itself, for example
some uses empty map, some new hashmap, but this method returned this
originally, so seemed like a good idea to keep it.
2015.10.19. 13:18 ezt írta ("Peter Farkas" <notifications@github.com>):

> In src/main/java/com/googlecode/objectify/impl/LoadTypeImpl.java
> <https://github.com/nbali/objectify/pull/1#discussion_r42359680>:
>
> > @@ -122,6 +123,10 @@
> >  		for (S id: ids)
> >  			keymap.put(this.<T>makeKey(id), id);
> >
> > +		if (keymap.isEmpty()) {
> > +			return Maps.newLinkedHashMap();
>
> Collections.emptyMap();? Or you want it to be writable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nbali/objectify/pull/1/files#r42359680>.
>